app/src/main/java/com/dessalines/thumbkey/keyboards/MessageEaseDE.kt

```diff
@@ -657,3 +658,5 @@ val MESSAGEEASE_DE_KEYBOARD_MODES: Map<KeyboardMode, KeyboardC> = mapOf(
     KeyboardMode.SHIFTED to MESSAGEEASE_DE_SHIFTED,
     KeyboardMode.NUMERIC to NUMERIC_KEYBOARD,
 )
+
+val MESSAGEEASE_DE_KEYBOARD_AUTOCAPITALIZERS: Autocapitalizers = arrayOf()
```

Comment:
Sorry to have to make you go through this work because its a lot of boilerplate... but now that there is yet another  "setting" for a keyboard outside of its keyboard definitions, I think we need to make a proper keyboard data class, that can hold this setting too. 

I also recommend we move away from using a `Map` of the different modes too ( and possibly that `KeyboardMode` enum)

So at the top of `Utils.kt`, add a data class that looks something like:

```kt
data class KeyboardDefinition {
  modes: {
    main: KeyboardC,
    shifted: KeyboardC,
    numeric:...
  },
  settings: {
    autoCapitalize: Array<AutoCapitalize>
   }
}
```

Reply:
I totally agree. Using a data class was my first idea, but it required some extra work. I will start working on this. Getting "rid" of autocap for "i" is the only thing keeping me from using the upstream version. I'm currently using my own debug build. 

Is it OK to rename keyboard layout files? I was thinking files and keyboard titles could start with country code, for easier discoverability. Now they are named `Type | Language | Version`, we could do `Language | Type | Version`. For instance `NOThumbKeyV1.kt`. Any thoughts?